uoftscrapers/scrapers/courses/__init__.py

```diff
@@ -124,15 +124,32 @@ def parse_course_html(course_id, html):
 
         # Things that don't appear on all courses
 
+        # If Campus is UTSG, find Breadths
         as_breadth = soup.find(id="u122")
         breadths = []
-        if as_breadth is not None:
+        if as_breadth is not None and campus == "UTSG":
             as_breadth = as_breadth.find_all("span", id="u122")[0] \
                 .get_text().strip()
             for ch in as_breadth:
                 if ch in "12345":
                     breadths.append(int(ch))
 
+        # If Campus is UTSC, find Breadths
```

Comment:
Might want to put this into an `elif` branch since it wouldn't make sense to run this at all if we already did UTSG above.